### ISSUE TITLE
Matrix transpose + inline comment

### ISSUE THREAD

**Issue by tdy** [daeyun/vim-matlab#6]
[`comment_pattern = re.compile(r"(^(?:[^'%]|'[^']*')*)(%.*)$")`](https://github.com/daeyun/vim-matlab/blob/841a09d/rplugin/python/vim_matlab/python_vim_utils.py#L9) gets hung up with single apostrophes.

**Raw code:**
```matlab
%%
A = sqrtm(B) \ C.'; % foo
Z = A * Y;          % bar
```
**Run code:**
```matlab
,A = sqrtm(B) \ C.'; % foo,Z = A * Y;
```

Not sure if there's a regex to handle comment markers + string apostrophes + transpose apostrophes.

Thought about switching [`','.join(lines)`](https://github.com/daeyun/vim-matlab/blob/841a09d/rplugin/python/vim_matlab/matlab_cli_controller.py#L24) to `"\n".join`, but that seems to create side effects like line continuation issues (in other code, not in this example).

**Comment by daeyun** [comment]
Thanks for the bug report. 👍✨  Patched in 1dcee6e. Did this fix the issue?

**Comment by tdy** [comment]
Nice, works on my end 👍
